emission/integrationTests/start_integration_tests.sh

```diff
@@ -6,9 +6,8 @@ cd /src/e-mission-server
 echo ${DB_HOST}
 if [ -z ${DB_HOST} ] ; then
     local_host=`hostname -i`
-    sed "s_localhost_${local_host}_" conf/storage/db.conf.sample > conf/storage/db.conf
-else
-    sed "s_localhost_${DB_HOST}_" conf/storage/db.conf.sample > conf/storage/db.conf
+    export DB_HOST=$local_host
+    echo "Setting db host environment variable to localhost"
```

Comment:
I don't think we need this environment variable set here anyway, since start_integration_tests.sh runs via a [Dockerfile ](https://github.com/e-mission/e-mission-server/blob/067158c404ecff6c3519335d4d74465eb93dcb4e/emission/integrationTests/Dockerfile)whose corresponding [compose](https://github.com/e-mission/e-mission-server/blob/067158c404ecff6c3519335d4d74465eb93dcb4e/emission/integrationTests/docker-compose.yml) has the variable hardcoded.

Reply:
I tested the server with and without the reload. When I removed the reload, one of the tests failed:
<img width="682" alt="Screenshot 2024-05-23 at 11 11 49 PM" src="https://github.com/e-mission/e-mission-server/assets/90212258/6eb2d0c7-6f2b-4467-9f52-d40bc79a7bcd">
This particular test calls the cfc_webapp.py. So, the reload is useful in this case!

Reply:
@nataliejschultz fair, but the integration tests are not the only way that people run the server. Developers run the server on their laptops to make changes as well. You need to now change the README to indicate that the environment variable should be set ...